outputs/elasticsearch/api.go

```
@@ -155,27 +155,27 @@ func (es *Elasticsearch) PerformRequest(conn *Connection, req *http.Request) ([]
 	resp, err := es.client.Do(req)
 	if err != nil {
 		// request fails
-		do_retry := false
+		doRetry := false
 		if isConnTimeout(err) || isConnRefused(err) {
 			es.connectionPool.MarkDead(conn)
-			do_retry = true
+			doRetry = true
 		}
-		return nil, do_retry, fmt.Errorf("Sending the request fails: %s", err)
+		return nil, doRetry, fmt.Errorf("Sending the request fails: %s", err)
 	}
 
 	if resp.StatusCode > 299 {
 		// request fails
-		do_retry := false
+		doRetry := false
 		if resp.StatusCode == http.StatusServiceUnavailable ||
 			resp.StatusCode == http.StatusGatewayTimeout {
 			// status code in {503, 504}
 			es.connectionPool.MarkDead(conn)
-			do_retry = true
+			doRetry = true
 		}
-		return nil, do_retry, fmt.Errorf("%v", resp.Status)
+		return nil, doRetry, fmt.Errorf("%v", resp.Status)
 	}
 
-	defer resp.Body.Close()
+	defer func() { _ = resp.Body.Close() }()
```

no real difference, but makes clear we are ignoring the error.

We could replace this by something like 'defer common.Closing(resp.Body)'.